### ISSUE TITLE
not able to setup the demo env

### ISSUE THREAD

**Issue by satyarox** [niklaushirt/ibm-aiops-deployer#105]
Hi @niklaushirt 

I did followed your steps and not able to get it succeeded 

Getting the below error in the pod install log

```
consolenotification.console.openshift.io/ibm-aiops-notification-warning created
Error from server (NotFound): routes.route.openshift.io "ibm-aiops-demo-ui" not found
Error from server (NotFound): configmaps "ibm-aiops-demo-ui-config" not found
consolenotification.console.openshift.io/ibm-aiops-notification-main created

```

![DEMOUI](https://github.com/niklaushirt/ibm-aiops-deployer/assets/43928676/776812b5-abbc-4ab8-9dd5-a9878c037d92)

**Comment by niklaushirt** [comment]
This is too vague, can you attach the complete installer log please?

**Comment by niklaushirt** [comment]
Should be fixed with #109

**Comment by satyarox** [comment]
[ibm-aiops-install-aiops-bxz5g-install.log](https://github.com/niklaushirt/ibm-aiops-deployer/files/14360264/ibm-aiops-install-aiops-bxz5g-install.log)
